src/test/java/com/amazon/opendistroforelasticsearch/security/http/proxy/HTTPExtendedProxyAuthenticatorTest.java

```diff
@@ -29,6 +29,8 @@
  */
 package com.amazon.opendistroforelasticsearch.security.http.proxy;
 
+import static org.hamcrest.MatcherAssert.*;
```

Comment:
nit: prefer `junit` over `hamcrest`.

Reply:
In this case (list comparison), I think that hamcrest is better than junit. 
Why should junit be prefered to hamcrest ?

Reply:
For consistency. Looking at the code it is hard to guess that `assertNull()` is from different library than `assertThat()`.